mix.exs

```diff
@@ -1,8 +1,8 @@
-defmodule NervesSystemRpi3.MixProject do
+defmodule CustomRpi3.MixProject do
```

Comment:
I believe you pushed this on accident and prob need to remove this commit

Reply:
Yup stupidly sent this request from the main branch apologies, should be resolved now?